releases/CHANGELOG-3.0.md

```diff
@@ -0,0 +1,205 @@
+## Breaking changes
+
+### Removed
+
+- The Tree View widget will no longer be a part of the Viewer by default
+- The Property Grid widget will no longer be a part of the Viewer by default
+- Selection Tools will no longer be a part of the Viewer by default
+- Measure Tools will no longer be a part of the Viewer by default
+- Navigation Tools will no longer be a part of the Viewer by default
+- The status bar items will no longer be a part of the Viewer by default
+
+All of these items can be easily added back via their respective UI Providers and initialized in the onIModelAppInit callback:
```

Comment:
wondering if we need to explicitly add links to the vcr pkgs

Reply:
The sample code has the necessary imports. I feel like that is sufficient?